### PR TITLE
New version: Git.Git version 2.40.1

### PR DESCRIPTION
**Pull request by raphael-zenhaeusern-legic**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103724)

### CODE REVIEW

File: manifests/g/Git/Git/2.40.1/Git.Git.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+PackageVersion: 2.40.1
+PackageLocale: en-US
+Publisher: The Git Development Community
+PublisherUrl: https://gitforwindows.org
+PublisherSupportUrl: https://github.com/git-for-windows/git/issues
+# PrivacyUrl:
+Author: Johannes Schindelin
+PackageName: Git
+PackageUrl: https://gitforwindows.org
+License: GNU General Public License version 2
```
Comment (BrandonWanHuanSheng) [inline manifests/g/Git/Git/2.40.1/Git.Git.locale.en-US.yaml:11]: The short form of License is GPL-2.0

File: manifests/g/Git/Git/2.40.1/Git.Git.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+LicenseUrl: https://raw.githubusercontent.com/git-for-windows/git/main/COPYING
+Copyright: Copyright (C) 1989, 1991 Free Software Foundation, Inc.
```
Comment (BrandonWanHuanSheng) [inline manifests/g/Git/Git/2.40.1/Git.Git.locale.en-US.yaml:13]: The copyright is Copyright (C) Free Software Foundation, Inc.

**Review by BrandonWanHuanSheng — CHANGES_REQUESTED** [review]
It looks like some data to make the manifest more completely.

**Comment by raphael-zenhaeusern-legic** [comment]
@BrandonWanHuanSheng adjusted. Though worth noting that there are around 2k manifests referencing GPL by its full name (including the previous releases of Git).

**Review by stephengillie — APPROVED** [review]

**Pull request merged**